addons/pronelauncher/XEH_preInit.sqf

```diff
@@ -0,0 +1,52 @@
+#include "script_component.hpp"
+
+ADDON = false;
+
+PREP_RECOMPILE_START;
+#include "XEH_PREP.hpp"
+PREP_RECOMPILE_END;
+
+[{!isNull (findDisplay 46)}, {
+    (findDisplay 46) displayAddEventHandler ["KeyDown", {
```

Comment:
This should be  separated into its own function.

Reply:
https://github.com/acemod/ACE3/blob/cd2619a4d14ef5aef70b3e2726d5a43f733800ed/addons/pronelauncher/functions/fnc_addKeyDownHandler.sqf

Reply:
better move handler code only to function, not handler adding

Reply:
IIRC, even though you are passing code, it gets turned back into a string and recompiled each time it is called?
So it's best to keep bis event handlers short 
`displayAddEventHandler ["KeyDown", {call FUNC(handleKeyDown)}]`

Reply:
>IIRC, even though you are passing code, it gets turned back into a string and recompiled each time it is called?
yep

Reply:
Changed with https://github.com/acemod/ACE3/blob/d70693b6d64cd052bb161116c527fc10ee57a9d1/addons/pronelauncher/XEH_preInit.sqf#L9